7-OOP-Geometry/Main.py

```diff
@@ -0,0 +1,16 @@
+from Circle import *
```

Comment:
Preferable way is Python 3.
See the link https://stackoverflow.com/questions/4142151/how-to-import-the-class-within-the-same-directory-or-sub-directory